addons/project/static/src/components/project_right_side_panel/project_right_side_panel.js

```diff
@@ -70,9 +70,9 @@ export class ProjectRightSidePanel extends Component {
     }
 
     formatMonetary(value, options = {}) {
-        const valueFormatted = formatMonetary(value, {
-            currencyId: this.currencyId,
+        const valueFormatted = formatFloat(value, {
```

Comment:
you can check the discussion i had with Vincent earlier in the review. But the issue was that 'formatMonetary' will always use the currency setting instead of the manual option given. It forced me to re-format the result of formatmonetary with either formatFloat or some js manipulation. I've discussed with vincent, and th ebest option we've come with was that.